packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -418,6 +423,9 @@ export default class SubmodeLayout extends Component<Signature> {
         margin-right: var(--boxel-sp);
         border-radius: var(--boxel-border-radius);
       }
+      .workspace-button:disabled {
+        cursor: default;
```

Comment:
Should this be built-into IconButton?

Reply:
yeah, good point

Reply:
fixed in https://github.com/cardstack/boxel/pull/1658/commits/ef6a48743cd14f1bb045db1b2f93067275052533